### ISSUE TITLE
Brute force Checklist.

### ISSUE THREAD

**Issue by vincentdavis** [biopython/biopython.github.io#38]
As suggested by Markus Piotrowski
Checklist:
1. Title
- The title in the markdown file may be changed into something more user-friendly (https://github.com/biopython/biopython.github.io/issues/22)
- Take care that the title is not repeated in the main text (like here: http://biopython.org/wiki/Documentation)

2. Is the text up-to-date? (Python versions etc)

3. Formatting
- Suggestion: Python/Biopython commands, keywords, module/function names should be formatted as inline-code with single backticks, e.g. `Bio.SeqIO`           ??? Or bold???
- If applicable, they can also serve as links to the respective Wiki page
- Code output and command-line examples should be formatted as block code with three backticks instead of several single line inline-code statements.
- Check block code for trailing white spaces, which may result in the addition of an (unnecessary) horizontal scrollbars as here: http://biopython.org/wiki/ACE_contig_to_alignment.

4. Code
- Code should be checked for a minimum of PEP8 compliance.
- The code should work (as it is) under Biopython 1.66
- Since we are recommending Python 3.5 as environment, the code should work under Python 3.5      ???

5. Links
- Check all links
- 'Repair' broken links
Try to find a link that's more likely to be stable. E.g. for papers I think that dx.doi.org references (or Pubmed references) are more stable than linking to a special page of the respective journal or private or institutional homepages. With http://www.crossref.org/SimpleTextQuery/ you can do a reverse doi lookup for a given paper.
- Links with anchors (.../some_page#jump_here) may have issues with upper/lowercase formatting (https://github.com/biopython/biopython.github.io/issues/13)
- Are the links up-to-date? E.g. in http://biopython.org/wiki/Getting_Started the Python Quick Reference links to Python 2.5

6. References
- Some pages used a PubMed plugin under MediaWiki to display references (https://github.com/biopython/biopython.github.io/issues/12). Since it's unlikely (?) to have a functional replacement, I would suggest to convert them to simple links.

7. RSS feeds
- As above, embedding an existing RSS feed seems hard (https://github.com/biopython/biopython.github.io/issues/4), again I would suggest replace with a link pointing to the RSS feed

**Comment by peterjc** [comment]
Yes, as with the main Tutorial and docstrings, try to make code examples work on both Python 2 and 3 (using simple print functions with one variable). If not easy, I think it is time to prefer Python 3 syntax but perhaps the example should not this explicit for the lingering Python 2 users.

**Comment by MarkusPiotrowski** [comment]
Internal links with anchors: See also #13
 
The correct format for anchored links **in the Wiki pages** is "all lower case with dashes", 
e.g. [http://biopython.org/wiki/Publications#**publications-from-2001**](http://biopython.org/wiki/Publications#publications-from-2001)

Each heading has such an anchor, so you can easily jump/link to any heading you like.
